triad/collections/fs.py

```diff
@@ -67,26 +67,36 @@ def _delegate(self, path) -> Tuple[FSBase, str]:
 
 class _FSPath(object):
     def __init__(self, path: str):
-        if path is None:
-            raise ValueError("path can't be None")
+        if path is None or path == "":
+            raise ValueError("path can't be None or empty")
         path = self._modify_path(path)
-        if path.startswith("file://"):
-            path = path[6:]
-        if path.startswith("/"):
+        # Windows
+        if path[0].isupper() or "\\" in path:
```

Comment:
`and`?

Reply:
I think _or_ because a path can be something like `\\temp\\a.csv`. It just ends up being treated as absolute.

Reply:
what about `TMP/file.txt`?